custom/icds_reports/utils/__init__.py

```diff
@@ -784,43 +784,43 @@ def person_is_beneficiary_column(beta):
 
 
 def wasting_moderate_column(beta):
-    return 'wasting_moderate_v2'
+    return 'wasting_moderate'
 
 
 def wasting_severe_column(beta):
-    return 'wasting_severe_v2'
+    return 'wasting_severe'
 
 
 def wasting_normal_column(beta):
-    return 'wasting_normal_v2'
+    return 'wasting_normal'
 
 
 def stunting_moderate_column(beta):
-    return 'zscore_grading_hfa_moderate'
+    return 'stunting_moderate'
 
 
 def stunting_severe_column(beta):
-    return 'zscore_grading_hfa_severe'
+    return 'stunting_severe'
 
 
 def stunting_normal_column(beta):
-    return 'zscore_grading_hfa_normal'
+    return 'stunting_normal'
 
 
 def current_month_stunting_column(beta):
-    return 'current_month_stunting_v2'
+    return 'current_month_stunting'
 
 
 def current_month_wasting_column(beta):
-    return 'current_month_wasting_v2'
+    return 'current_month_wasting'
 
 
 def hfa_recorded_in_month_column(beta):
-    return 'zscore_grading_hfa_recorded_in_month'
+    return 'height_measured_in_month'
```

Comment:
Is not this moving the focus from zscore question to height_child question.
Along with that, it would make the numerator working on zscore property and denominator working on height_child. 
Is that fine. Saying because we have seen lots of problems between these two like one is filled and one is not.

Reply:
Yeah, offlined with Nitish about that and he thinks this is a positive change. And again, this is how it used to be, I suspect our movement away from that behavior was accidental.

Reply:
I see. ok ok . sounds good.